operators/aqua/2022.4.15/metadata/annotations.yaml

```
@@ -0,0 +1,10 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: 2022.4.15
```

```suggestion
  operators.operatorframework.io.bundle.channel.default.v1: 2022.4.0
```

@adishaull this is the cause of the validation error: this operator has no `2022.4.15` channel.

---

Thanks @mporrato , do to care to explain what is this channel, each version we using 
  operators.operatorframework.io.bundle.channel.default.v1: 2022.4.0
  operators.operatorframework.io.bundle.channels.v1: 2022.4.0

i revert to this configuration, but it's still failing